packages/flutter/lib/src/material/data_table.dart

```diff
@@ -467,6 +469,11 @@ class DataTable extends StatelessWidget {
   static const Color _grey100Opacity = Color(0x0A000000); // Grey 100 as opacity instead of solid color
   static const Color _grey300Opacity = Color(0x1E000000); // Dark theme variant is just a guess.
 
+  /// The width of the divider that appears between [TableRow]s.
+  /// Must be non-null and greater than or equal to zero.
```

Comment:
```suggestion
  /// 
  /// Must be non-null and greater than or equal to zero.
```

Nit: Dart docs specify a blank line should follow each paragraph, and that the first sentence should be it own paragraph.